packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/definecallback.md

```diff
@@ -0,0 +1 @@
+In this iOS quickstart, we don't define an explicit callback view because it is required only if the system needs to launch the app or open it from the background. For this redirect case, the app is always in the foreground as the Okta SDK opens a view that's a part of the app.
```

Comment:
This is a semantically incorrect statement. You don't define a callback view, it's a callback URI. The app determines what to show when the system opens the app by sending a callback URI. Also, I really don't like the use of "we" both because it's usually not clear to whom it refers (Okta, Okta and the developer, ...), and the tone is too familiar (and different from the rest of the writing.) Bottom line, I suggest this wording:

A callback URI is required only for the system opening the app from a URL, such as one sent with a notification. This sample is always in the foreground so no callback URI is required.

Reply:
Much better, thanks. Updated.